tools/incubator.js

```diff
@@ -209,6 +209,22 @@ const Incubator = new Lang.Class({
             basin_manifest['content'].push(this._import_video(asset));
         });
 
+
+        /* Once we're done, look over all the content in the manifest and
+         * set the parent key on each metadata struct. */
+        let manifest_table = {};
+        basin_manifest.content.filter(metadata => metadata['@id'])
+                              .forEach(metadata =>
```

Comment:
Indentation style should match the rest of the code.

Reply:
I can't really see much precedent in the code for how run-on method calls are supposed to be handled so I've left this as-is for now (without the hanging parens, since I figured perhaps they where what was controversial).

Have you got an example in mind?